kuma/wiki/models.py

```diff
@@ -920,6 +923,9 @@ def save(self, *args, **kwargs):
             # If this is a translation without a topic parent, try to get one.
             self.acquire_translated_topic_parent()
 
+        # Set the shortened URL (e.g. bit.ly).
+        self.set_share_url(save=False)
```

Comment:
I thought we'd run this as a Celery task to be honest to not make document saving slower and potentially blocking on a call to Bitly's API.